api/src/main/java/uk/gov/dhsc/htbhf/claimant/entitlement/BackDatedCycleEntitlementCalculator.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.dhsc.htbhf.claimant.entitlement;
+
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.min;
+
+/**
+ * Calculates the number of back dated vouchers a claimant is entitlement to.
+ * The new children that have been matched to the expected due date and the expected due date are used to
+ * calculate the difference between the number of vouchers that were received for pregnancy and
```

Comment:
The expected due date, and the birthdays of children resulting from the pregnancy are used to
 * calculate the difference between the number of vouchers

Reply:
Updated.